crawl-ref/source/describe.cc

```diff
@@ -3779,6 +3779,7 @@ static string _flavour_base_desc(attack_flavour flavour)
         { AF_REACH_TONGUE,      "deal extra acid damage" },
         { AF_WEAKNESS,          "cause weakness" },
         { AF_BARBS,             "embed barbs" },
+        { AF_SPIDER,            "summon a spider" },
```

Comment:
Flavor: would this be more fun as 'spawn a spider', or would it be too misleading re: summoned status?